ext/reflection/tests/ReflectionProperty_isWritable.phpt

```diff
@@ -0,0 +1,90 @@
+--TEST--
+Test ReflectionProperty::isWritable()
+--FILE--
+<?php
+
+class A {
```

Comment:
Why this test does not check protected properties from parent/child classes?

Reply:
Because the impl uses the same code for checking scope as read, so it's somewhat redundant.

Reply:
Tests should not assume any implementation details, because implementation might change later, but the tests should detect regression no matter how things are implemented.